roles/kube-install/tasks/main.yml

```diff
@@ -70,7 +70,9 @@
     - kubeadm{{ kube_version_parameter }}
     - kubernetes-cni
 
-
+- name: Optionally include playbook for binary install
+  include: binary_install.yml
```

Comment:
s/include/import_playbook/

Reply:
This is dependent on us wanting Ansible 2.4 to be the starting minimum version.

Reply:
I'm gonna stick with `include:` only because we've been using it globally (earliest reference of our use [about 7 months ago](https://github.com/redhat-nfvpe/kube-centos-ansible/blame/master/roles/kube-install/tasks/main.yml#L82))